sbi/inference/snpe/snpe_c.py

```diff
@@ -93,7 +93,7 @@ def _log_prob_proposal_posterior(
         Args:
             theta: Batch of parameters θ.
             x: Batch of data.
-            masks: Whetherto retrain with prior loss (for each prior sample).
+            masks: Whether to retrain with prior loss (for each prior sample).
```

Comment:
the docstring is correct, but isnt it more than that? it is a boolean tensor marking prior samples with true to then train them with prior loss. therefore: "masks: mask that is `True` for prior samples in the batch in order to train them with prior loss (instead of proposal loss)." or so?